source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,107 @@
+
+Direct Debit payments are an asynchronous payment method. When a user
+completes a Direct Debit payment process using GOV.UK Pay, the platform makes
+a request to their bank to set up the Direct Debit mandate and transfer the
+funds. It takes several days to confirm the success of the payment. 
+
+## One-off payments using Direct Debit
+
+To set up a Direct Debit mandate to take a single payment:
+
+* create an API token linked to your Direct Debit service
+* follow the instructions in the [‘Payment Flow’ section](/payment_flow)
+
+You cannot take further payments from users using this method without creating
+a new payment. This requires users to re-enter their payment details.
+
+## Recurring payments using Direct Debit
+
+You can create recurring Direct Debit payments using the POST /v1/agreements
+endpoint. 
+
+### On-demand payment agreement
+
+An on-demand (or “variable”) agreement allows you to collect payments of any
+amount at any time. For example, you may want to collect: 
+
+* money each time a service is provided
+* payments where the amounts vary according to service usage
+* regular payments where you do not know in advance how much each payment will be
+
+You are responsible for initiating on-demand payments using the GOV.UK Pay API. 
+
+Set up a recurring Direct Debit payment
+
+1. Create an agreement using `POST /v1/agreements/`,
+specifying:
+
+* a return_url where GOV.UK Pay will direct the user when they have finished
+setting up the agreement
+* the `agreement_type` (`on demand`)
+* a reference (optional) - your reference for this agreement 
+
+The response contains a `next_url`.
+
+2. Direct the user to the `next_url`.
+
+3. The user provides their payment details. For example, account number
+and sort code for Direct Debit. GOV.UK Pay makes an API call to our Direct
+Debit supplier to verify account details.
+
+4. GOV.UK Pay presents the user with a confirmation screen which presents
+their account details. The user grants their consent by selecting
+**Confirm**.  GOV.UK Pay sends an email to the user to confirm that their
+mandate has been set up. If the mandate subsequently fails, the GOV.UK Pay team
+will send an email to update the user that their mandate could not be set up.
+
+5. GOV.UK Pay securely passes the user's details on to the Direct Debit
+provider, who submit a request to BACS to set up a mandate.
+
+6. GOV.UK Pay returns the user to your service using the `return_url` from 
+Step 1.
+
+7. Call `GET /v1/agreements/<AGREEMENT-ID>` to check the status of the
+agreement (in order to display an appropriate screen). For a Direct Debit
+payment, this will initially be in a ‘pending’ state until the bank confirms
+the mandate. The confirmation process typically takes 3-4 days.
+
+8. When the GOV.UK Pay team receives confirmation that the mandate is
+confirmed, the GOV.UK Pay admin tool will update the transaction status. You
+can check the status of the transaction by calling the events endpoint.
```

Comment:
What's the `events` endpoint, in full (and again will it be on Gelato)?

Reply:
the events endpoint will show a service what's happened with a mandate or payment - we don't have something like this yet since card payments are synchronous, but DD is asynchronous so we need to give services a way to find out what's happened with a transaction instead of purely relying on gov.uk pay updating the transaction. the endpoint isn't fully developed yet. once it is, it'll go in gelato.

Reply:
OK, do we want to include that in these docs?